core/resources/shaders/point.fs

```diff
@@ -48,8 +48,16 @@ void main(void) {
             float c2 = circle(uv, vec2(0.0), circleRadiusOut);
             color = vec4(vec3(c1) * v_color.rgb, c2 * v_alpha * v_color.a);
         #else
+            // premultiplied texture color
             vec4 texColor = texture2D(u_tex, v_texcoords);
-            color = vec4(texColor.rgb * v_color.rgb, v_alpha * texColor.a * v_color.a);
+            //color = vec4(texColor.rgb * v_color.rgb, texColor.a * v_color.a);
+
+            // Correct? v_color as multipler - to say how much of each texture channel is used?
+            color = texColor * v_color;
+            color *= v_alpha;
```

Comment:
Yeah I guess what I mean is that transparency is not something that is usually used everywhere where a texture is used, mostly the case for POIs/icons. But for consistency we could change the blend mode.

I could see someone wanting to apply a Mapzen texture on all of building Manhattan, and he could get quite disappointed!
![screen shot 2016-03-03 at 11 28 28 am](https://cloud.githubusercontent.com/assets/7061573/13501105/5ad1ba5e-e133-11e5-83fd-fe815d9803af.png)